labs/07_troubleshooting_ops.md

```diff
@@ -54,45 +54,43 @@ KUBERNETES_PORT_53_TCP=tcp://172.30.0.1:53
+The log files for a pod can be displayed in the web console as well as in the cli:
 
 ```
 $ oc logs [POD]
 ```
-Der Parameter `-f` bewirkt analoges Verhalten wie `tail -f`
-
-Befindet sich ein Pod im Status **CrashLoopBackOff** bedeutet dies, dass er auch nach wiederholtem Restarten nicht erfolgreich gestartet werden konnte. Die Logfiles können auch wenn der Pod nicht läuft mit dem folgenden Befehl angezeigt werden.
+The `-f` parameter has the same behavior as `tail -f`
+If a pod has the status **CrashLoopBackOff** this means it could not be started succesfully even after repeated restarts. The logfiles still can be displayed even if the pod is not running with the following command:
 
  ```
 $ oc logs -p [POD]
 ```
 
-Mit OpenShift wird ein EFK (Elasticsearch, Fluentd, Kibana) Stack mitgeliefert, der sämtliche Logfiles sammelt, rotiert und aggregiert. Kibana erlaubt es Logs zu durchsuchen, zu filtern und grafisch aufzubereiten. Weitere Informationen und ein optionales LAB finden sie [hier](../additional-labs/logging_efk_stack.md).
-
+With Openshift an EFK (Elasticsearch, Fluentd, Kibana) stack is delivered, which collects, rotates and aggregates all log files. Kibana allows logs to be searched, filtered and graphically edited. For more information and an optional Lab see [here](../additional-labs/logging_efk_stack.md).
 
 ## Task: LAB7.3 Port Forwarding
 
-OpenShift 3 erlaubt es, beliebige Ports von der Entwicklungs-Workstation auf einen Pod weiterzuleiten. Dies ist z.B. nützlich, um auf Administrationskonsolen, Datenbanken, usw. zuzugreifen, die nicht gegen das Internet exponiert werden und auch sonst nicht erreichbar sind. Im Gegensatz zu OpenShift 2 werden die Portweiterleitungen über dieselbe HTTPS-Verbindung getunnelt, die der OpenShift Client (oc) auch sonst benutzt. Dies erlaubt es auch dann auf OpenShift 3 Platformen zuzugreifen, wenn sich restriktive Firewalls und/oder Proxies zwischen Workstation und OpenShift befinden.
+OpenShift 3 allows us to forward any port from our workstation to the pod. This is usefull to use adminstrationconsoles, databases and so on which should not be exposed towards the internet. In contrast to Openshift 2 the portforwarding is tunneled through the same HTTPS-connection as the Openshift Client (`oc`) uses. This is usefull if there are restrictive Firewalls and/or Proxies between your Workstation and Openshift.
 
-Übung: Auf die Spring Boot Metrics aus [Lab 4](04_deploy_dockerimage.md) zugreifen.
+Excercise: Acces the Spring Boot Metrics from [Lab 4](04_deploy_dockerimage.md).
 
 ```
 oc get po --namespace="[USER]-dockerimage"
 oc port-forward example-spring-boot-1-xj1df 9000:9000 --namespace="[USER]-dockerimage"
 ```
 
-Nicht vergessen den Pod Namen an die eigene Installation anzupassen. Falls installiert kann dafür Autocompletion verwendet werden.
+Don't forgett to change the Pod Name accordingly. If installed you can use the TAB-completion.
```

Comment:
```suggestion
Don't forget to change the pod name accordingly. If installed you can use TAB-completion.
```

Reply:
I believe, it should say, "tab completion" (no dash, no capital letters).

Reply:
updated